# Goal
int8 area

---
# Result
bool arrived_goal

---
# Feedback
string current_coords